core/src/main/java/com/sequenceiq/cloudbreak/fluent/FluentConfigService.java

```diff
@@ -26,78 +34,127 @@
 
     private static final String S3_PROVIDER_PREFIX = "s3";
 
+    private static final String WASB_PROVIDER_PREFIX = "wasb";
+
     private static final String DEFAULT_PROVIDER_PREFIX = "stdout";
 
     public FluentConfigView createFluentConfigs(Stack stack, Telemetry telemetry) {
+        final FluentConfigView.Builder builder = new FluentConfigView.Builder();
+        boolean fluentEnabled = false;
         if (telemetry != null && telemetry.getLogging() != null) {
             Logging logging = telemetry.getLogging();
             String clusterName = stack.getCluster().getName();
             String platform = stack.getCloudPlatform();
-            StackType stackType = stack.getType();
+            String storageLocation = logging.getStorageLocation();
 
-            String clusterType = CLUSTER_TYPE_DISTROX;
-            if (StackType.DATALAKE.equals(stackType)) {
-                clusterType = CLUSTER_TYPE_SDX;
-            }
-            String s3bucket = null;
-            String s3LogFolderName = null;
-            boolean fluentEnabled = false;
-            String providerPrefix = DEFAULT_PROVIDER_PREFIX;
+            String clusterType = StackType.DATALAKE.equals(stack.getType()) ? CLUSTER_TYPE_SDX : CLUSTER_TYPE_DISTROX;
```

Comment:
Not a big deal but do we really want to name data lake folders as `"sdx"`? According to the [CDP Glossary](https://docs.google.com/document/d/1SyXqp0fhjhEmwLCSeScon-i3Lf7HKnwRpDkgU-dKPHI/edit), "Data Lake Cluster" is official term used in marketing and UI labels (even though internal CB API endpoints and CB CLI still refer to them as "sdx clusters").

Reply:
then it probably can be instead of distrox or sdx like: datalake and datahub